### PR TITLE
Update man page example and links, add --help option again

### PR DESCRIPTION
**Pull request by timopollmeier**

### CODE REVIEW

File: doc/gvmd.8 [hunk 1]
```diff
@@ -179,10 +182,16 @@ SIGHUP causes gvmd to rebuild the database with information from the Scanner (op
...
-.SH MORE INFORMATION ABOUT THE OPENVAS PROJECT
-The canonical places where you will find more information about the OpenVAS project are: \fBhttp://www.openvas.org/\f1 (Official site) \fBhttp://wald.intevation.org/projects/openvas/\f1 (Development Platform) 
+.SH MORE INFORMATION ABOUT GREENBONE VULNERABILITY MANAGEMENT
+The canonical places where you will find more information about the Greenbone Vulnerability Manager are: 
+
+\fBhttps://community.greenbone.net\f1 (Community portal) 
+
+\fBhttps://github.com/greenbone\f1 (Development Platform) 
+
+\fBhttps:/greenbone.net\f1 (Greenbone website)
```
Comment (cfi-gb) [inline doc/gvmd.8]: ``https:/greenbone.net`` -> ``https://greenbone.net``

File: doc/gvmd.8.xml [hunk 2]
```diff
@@ -405,23 +412,29 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
...
+    <p>
+      <url href="https://community.greenbone.net"/>
+        (Community portal)
+    </p>
+    <p>
+      <url href="https://github.com/greenbone"/>
         (Development Platform)
     </p>
+    <p>
+      <url href="https:/greenbone.net"/>
```
Comment (cfi-gb) [inline doc/gvmd.8.xml]: ``https:/greenbone.net`` -> ``https://greenbone.net``

File: doc/gvmd.html [hunk 3]
```diff
@@ -394,26 +401,32 @@ <h2>SEE ALSO</h2>
...
+    <p>
+      <a href = "https://community.greenbone.net">https://community.greenbone.net</a>
+        (Community portal)
+    </p>
+    <p>
+      <a href = "https://github.com/greenbone">https://github.com/greenbone</a>
         (Development Platform)
     </p>
+    <p>
+      <a href = "https:/greenbone.net">https:/greenbone.net</a>
```
Comment (cfi-gb) [inline doc/gvmd.html]: ``https:/greenbone.net`` -> ``https://greenbone.net``

File: doc/gvmd.8.xml (hunk 2 above)
Comment (timopollmeier) [reply doc/gvmd.8.xml]: I've fixed this in 30167e42be23fa6c218abc80964f26206c36a038

**Pull request merged**